pkg/kubernetes/client_test.go

```diff
@@ -100,6 +100,26 @@ func TestCreateLoadTestCRNoLoadTest(t *testing.T) {
 	assert.Error(t, deleteErr)
 }
 
+func TestGetLoadTestsByLabel(t *testing.T) {
```

Comment:
Let's add a negative test as well, to cover both cases.

Reply:
I couldn't find any way to test negative for this function in fact, because it doesn't do anything but just listing. If you can see `TestGetLoadTest`, there is also no negative test for this.